roles/etcd/defaults/main.yaml

```diff
@@ -27,13 +27,13 @@ r_etcd_common_etcdctl_command: "{{ etcdctl_dict[r_etcd_common_etcd_runtime] }}"
 # etcd server vars
 etcd_conf_dir: '/etc/etcd'
 etcd_conf_file: "{{ etcd_conf_dir }}/etcd.conf"
-etcd_ca_file: "{{ etcd_conf_dir }}/ca.crt"
-etcd_cert_file: "{{ etcd_conf_dir }}/server.crt"
+etcd_ca_file: "{{ etcd_conf_dir }}/{{ etcd_cert_prefix }}ca.crt"
```

Comment:
nit: add a comment to mention that this prefix is empty by default and could be overridden in other playbooks